### PR TITLE
rename classes AutomaticFilter and ManualFilter

### PR DESCRIPTION
**Pull request by Walid-Sahnoun**
Signed-off-by: walid-sahnoun <walid.sahnoun@rte-france.com>

### CODE REVIEW

**Review by FranckLecuyer — CHANGES_REQUESTED** [review]

File: src/main/java/org/gridsuite/filter/server/dto/AbstractFilter.java [hunk 1]
```diff
@@ -27,8 +27,8 @@
 )
 @JsonSubTypes({//Below, we define the names and the binding classes.
     @JsonSubTypes.Type(value = ScriptFilter.class, name = "SCRIPT"),
-    @JsonSubTypes.Type(value = AutomaticFilter.class, name = "AUTOMATIC"),
-    @JsonSubTypes.Type(value = ManualFilter.class, name = "MANUAL")
+    @JsonSubTypes.Type(value = CriteriaFilter.class, name = "AUTOMATIC"),
```
[the remark this answers is not in the document]
Comment (Walid-Sahnoun) [reply src/main/java/org/gridsuite/filter/server/dto/AbstractFilter.java:30]: ok

File: src/main/java/org/gridsuite/filter/server/dto/AbstractFilter.java [hunk 2]
```diff
@@ -27,8 +27,8 @@
...
+    @JsonSubTypes.Type(value = IdentifierListFilter.class, name = "MANUAL")
```
[the remark this answers is not in the document]
Comment (Walid-Sahnoun) [reply src/main/java/org/gridsuite/filter/server/dto/AbstractFilter.java:31]: ok

File: src/main/java/org/gridsuite/filter/server/dto/IdentifierListFilter.java [hunk 3]
```diff
@@ -28,15 +28,15 @@
 @Schema(description = "Manual Filters", allOf = AbstractFilter.class)
```
[the remark this answers is not in the document]
Comment (Walid-Sahnoun) [reply src/main/java/org/gridsuite/filter/server/dto/IdentifierListFilter.java:28]: yes i will do

File: src/main/java/org/gridsuite/filter/server/repositories/IdentifierListFilterRepository.java [hunk 4]
```diff
@@ -13,5 +13,5 @@
  * @author Seddik Yengui <seddik.yengui at rte-france.com>
  */
 
-public interface ManualFilterRepository extends FilterRepository<ManualFilterEntity> {
+public interface IdentifierListFilterRepository extends FilterRepository<ManualFilterEntity> {
```
[the remark this answers is not in the document]
Comment (Walid-Sahnoun) [reply src/main/java/org/gridsuite/filter/server/repositories/IdentifierListFilterRepository.java:16]: well, in the User story only dto and jpa to change
maybe see that with the tech leads ?

**Review by FranckLecuyer — APPROVED** [review]

**Review by archendel — APPROVED** [review]

**Review by dbraquart — CHANGES_REQUESTED** [review]
derby.log must not be commited

**Review by dbraquart — APPROVED** [review]

**Pull request merged**